docs/openapi/index.yaml

```diff
@@ -0,0 +1,15 @@
+openapi: 3.0.2
+tags:
+  - name: status
+info:
+  version: 0.0.1
+  title: TTADP
+  contact:
+    email: josh@adhocteam.us
+servers:
+  - url: http://localhost:8080
+  - url: http://backend:8080
```

Comment:
Should this be configurable for different environments?

Reply:
Good catch, I'll add the dev environment to this list. We will want to add staging and prod once we know their URLs